src/navigation/config.js

```diff
@@ -128,14 +130,15 @@ export const sendConfirmationSheetConfig = {
 };
 
 export const explainSheetConfig = {
-  options: ({ route: { params = {} } }) => {
+  options: ({
+    route: { params = { network: networkInfo[networkTypes.mainnet].name } },
```

Comment:
curious to know why you didn't go with `networkInfo.mainnet.name`

Reply:
mainly just following how we're using elsewhere which seems to follow this pattern

Reply:
yes we should use our enums / consts (such as networkTypes) so as to reduce potential errors (say if we needed to rename these later)